### ISSUE TITLE
The primary sidebar will close automatically after a few seconds when opening the first time after installing this extension on macOS

### ISSUE THREAD

**Issue by singularitti** [davila7/code-gpt-docs#95]
After installing this extension, I discovered that whenever I start VSCode on macOS and open the primary sidebar (which shows extensions, history, files, etc.), it automatically closes after a few seconds for no apparent reason. However, when I open the primary sidebar again, it remains open and does not automatically close. This behavior only occurs the first time I open it. Using extension bisection, I figured out that this extension was causing the problem. After uninstalling it, everything worked fine.

VSCode info:

```
Version: 1.77.0
Commit: 7f329fe6c66b0f86ae1574c2911b681ad5a45d63
Date: 2023-03-29T09:55:48.214Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin arm64 22.4.0
Sandboxed: No
```

Extension version: 2.1.3

**Comment by zohaib87** [comment]
I have the same issue but in Windows 10. When I disabled this extension everything goes to normal.

**Comment by frankyueh** [comment]
I also have this same issue on Linux Ubuntu.

**Comment by JoelVenable** [comment]
@davila7 It's hard to show in a screenshot.  When launching VS code, after a few seconds the left pane simply disappears, as if I had closed it. For me, this behavior is new since installing this extension.  

This behavior is also exhibited when reloading the window from the command pallette.

Another (related?) issue is that when I reopen the pane, whatever tab I was on has been replaced by the CodeGPT pane.  I would expect it to be whatever I had previously selected.

**Comment by zohaib87** [comment]
Check this, the explorer closes after I open the project in VSCode. but when I disable the extension it goes to normal.

![Ai22HmNY2o](https://user-images.githubusercontent.com/10462023/229345891-40a68eef-320f-4548-a17d-b20caf0ba6b0.gif)

**Comment by rogadev** [comment]
This, combined with the dizzying scroll effect when generating text (#99), was enough to make me uninstall the extension today. :( 

```
Version: 1.77.0 (user setup)
Commit: 7f329fe6c66b0f86ae1574c2911b681ad5a45d63
Date: 2023-03-29T10:02:16.981Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: No
```

**Comment by davila7** [comment]
@singularitti @zohaib87 @frankyueh @JoelVenable @rogadev 

It is already solved in the new version 2.1.4 💪
We are doing the last tests to publish the version.
Thanks for the information.

**Comment by davila7** [comment]
The new version 2.1.4 is now available
